### PR TITLE
Fix icons on App page in FF

### PR DESCRIPTION
**Pull request by rullzer**
For #900
- [x] Filter not working
- [x] No scaling

Once done:
- [x] Check Chrome
- [x] Check FF
- [x] Check IE/Edge
- [x] Check Safari
- [x] Check opera

### CODE REVIEW

File: settings/js/apps.js [hunk 1]
```diff
@@ -234,6 +234,11 @@ OC.Settings.Apps = OC.Settings.Apps || {
 
 	imageUrl : function (url, appfromstore) {
 		var img = '<svg width="72" height="72" viewBox="0 0 72 72">';
+		// weird hack because in Firefox everything is 4 times as big (doubled in width and doubled in height)
+		// TODO: replace this with a proper solution
```
Comment (blizzz) [inline settings/js/apps.js]: can we have todos as issues instead, pls?

Comment (MorrisJobke) [inline settings/js/apps.js]: I will create it

Comment (MorrisJobke) [inline settings/js/apps.js]: #908

**Comment by rullzer** [comment]
Fine by me for now :+1:

**Comment by rullzer** [comment]
It seems it is because a lot of icons are missing a `viewBox` attribute. So browsers just say sorry but I can't scale that.
I pushed some updated icons to test.

**Comment by MorrisJobke** [comment]
I tested this now in Edge, IE11, IE10, Safari and it works just fine 👍

**Comment by MorrisJobke** [comment]
@karlitschek We need this backported (and it would be best to have this in before the RC - looking towards @LukasReschke 😉 )

**Comment by MorrisJobke** [comment]
I tested this with Opera too and it works there :)

**Comment by karlitschek** [comment]
nice. please backport

**Comment by LukasReschke** [comment]
LGTM and world doesn't explode 👍

**Comment by MorrisJobke** [comment]
@karlitschek for all the backports above 🙈 It's mostly only adding a `viewBox="0 0 32 32"` to the SVGs because otherwise FF and IE doesn't scale SVGs at all. In other browsers this is the default behavior.

Then the app management looks nice for all shipped apps.

**Pull request merged**